client/packages/common/src/ui/components/navigation/ListOptions/ListOptions.tsx

```diff
@@ -64,12 +66,7 @@ export const ListOptions = ({
           <ListItem
             sx={{ padding: '5px 0px', cursor: 'pointer' }}
             onClick={() => onClick(option.id)}
-            ref={
-              option.id === currentId
-                ? l =>
-                    l?.scrollIntoView({ behavior: 'smooth', block: 'center' })
-                : null
-            }
+            ref={option.id === currentId ? scrollRef : null}
```

Comment:
The `scrollRef` gets applied to the element containing the current selection, which changes. I've had to add a small time delay to the `scrollIntoView` method to ensure that the ref has been applied to the *new* selection before scrolling to it. Without that, it scrolls to the previously select (so it's off by one) as it starts scrolling before the re-render has changed the ref assignment.

I don't love it, but it seems the most straightforward for this particular situation.